lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -713,34 +715,33 @@ def process_parameters(self, params):
         else:
             if not isinstance(params['user_ids'], list):
                 raise ValueError('Variable user_ids' + ' must be a list.')
-	if 'kbasetest' in params['user_ids']:
-		params['user_ids'].remove('kbasetest')
+        if 'kbasetest' in params['user_ids']:
+            params['user_ids'].remove('kbasetest')
 
         if not params.get('epoch_range', None) is None:
             start_time, end_time = params['epoch_range']
-	    if (not start_time is None and not end_time is None):
-		start_time = _convert_to_datetime(start_time)
-		end_time = _convert_to_datetime(end_time)
-		params['minTime'] = _unix_time_millis_from_datetime(start_time)
-		params['maxTime'] = _unix_time_millis_from_datetime(end_time)
-	    elif (not start_time is None and end_time is None):
-		start_time = _convert_to_datetime(start_time)
-		end_time = start_time + datetime.timedelta(hours=48)
-		params['minTime'] = _unix_time_millis_from_datetime(start_time)
-		params['maxTime'] = _unix_time_millis_from_datetime(end_time)
-	    elif (start_time is None and not end_time is None):
-		end_time = _convert_to_datetime(end_time)
-		start_time = end_time - datetime.timedelta(hours=48)
-		params['minTime'] = _unix_time_millis_from_datetime(start_time)
-		params['maxTime'] = _unix_time_millis_from_datetime(end_time)
-        else: #set the most recent 48 hours range
+            if (start_time is not None and end_time is not None):
+                start_time = _convert_to_datetime(start_time)
+                end_time = _convert_to_datetime(end_time)
+                params['minTime'] = _unix_time_millis_from_datetime(start_time)
+                params['maxTime'] = _unix_time_millis_from_datetime(end_time)
+            elif (start_time is not None and end_time is None):
```

Comment:
@qzzhang Same here. Please verify the logic is correct. Also you should probably do `elif not start_time and end_time`.